### DISCUSSION TITLE
公式和目录索引

### DISCUSSION THREAD

**Discussion by MOWAGN** [tamlok/vnote#2157] [Q&A]
经常会出现目录点击后无法跳转对应内容和公式无法显示的问题，本人使用环境为WIN11，VNote为3.13.0，公式显示功能是需要网络支持的吗？无网状态是不是不能将正常的Latex公式自动转换为公式显示？

**Comment by tamlok** [comment]
https://vnotex.github.io/vnote/zh_cn/#!docs/%E7%94%A8%E6%88%B7/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98.md

新版的话，直接在设置里面设置即可。

OpenGL也设置一下。

**Reply by MOWAGN** [reply]
我试试，谢谢大佬
